dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/CacheHashMap.java

```
@@ -139,103 +139,109 @@
+                    if (trace && tc.isDebugEnabled())
+                        tcInvoke(cacheStoreService.tcCacheManager, "createCache", attrCacheName, config);
+
+                    sessionAttributeCache = cacheStoreService.cacheManager.createCache(attrCacheName, config);
+                } catch (CacheException x) {
+                    create = false;
+                    if (trace && tc.isDebugEnabled()) {
+                        tcReturn(cacheStoreService.tcCacheManager, "createCache", x);
+                        tcInvoke(cacheStoreService.tcCacheManager, "getCache", attrCacheName, "String", "byte[]");
+                    }
+                    sessionAttributeCache = cacheStoreService.cacheManager.getCache(attrCacheName, String.class, byte[].class);
+                    if (sessionAttributeCache == null)
+                        throw x;
                 }
-                sessionAttributeCache = cacheStoreService.cacheManager.getCache(attrCacheName, String.class, byte[].class);
-                if (sessionAttributeCache == null)
-                    throw x;
             }
-        }
 
-        tcSessionAttrCache = "AttrCache" + Integer.toHexString(System.identityHashCode(sessionAttributeCache));
-        if (trace && tc.isDebugEnabled())
-            tcReturn(cacheStoreService.tcCacheManager, create ? "createCache" : "getCache", tcSessionAttrCache, sessionAttributeCache);
+            tcSessionAttrCache = "AttrCache" + Integer.toHexString(System.identityHashCode(sessionAttributeCache));
+            if (trace && tc.isDebugEnabled())
+                tcReturn(cacheStoreService.tcCacheManager, create ? "createCache" : "getCache", tcSessionAttrCache, sessionAttributeCache);
 
-        cacheStoreService.configureMonitoring(attrCacheName);
+            cacheStoreService.configureMonitoring(attrCacheName);
+        } catch(Exception ex) {
+            FFDCFilter.processException(ex, "com.ibm.ws.session.store.cache.CacheHashMap.cacheInit", "241");
```

Why FFDCIgnore Exception, only to manually FFDC it with no added information?